stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<blockquote>The types of glass represented in the sample are: Window Float, Window Non-Float, Vehicle Float, Container, Tableware, and Headlamp. There are no Vehicle Non-Float types represented in the data.</blockquote><p> </p><p>As a complete outsider to this field, I don't really know what float means in this case. If that's important, I'd suggest adding a short sentence explaining that. I also don't think it's necessary to mention that Vehicle Non-Float types aren't represented.</p><p> </p><blockquote>There are three main research questions I wish to answer in my machine learning classification problem:</blockquote><blockquote>  1) What can Python's Matplotlib graphing capabilities reveal about the similarities and/or dissimilarities between the artifact, clear facsimile, and the glass samples in the 1987 database?</blockquote><blockquote>2) What do the correlations between elements for the different types of glass in the 1987 database reveal about late 20th century glassmaking, as compared with early 20th century glassmaking?</blockquote><blockquote>3) Using machine learning to train a model on the 1987 database, could that model be used to classify the type (container) of one or both of the older samples unseen by the model?</blockquote><p> </p><p>It might be worth adding a few sentences after the research questions explaining why they are interesting/unusual</p><p> </p><p>(apologies for the copy-pasting, I'm not sure how to comment on specific sentences rather than whole sections)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Hi Eirini, Thanks for these comments. I have been taken up with Python course work and I apologize for the lateness of my reply. Jennifer has made a similar point about explaining these glass types. Distinguishing them and defining them is important. I will review my sources and fill in this missing information.  </p><p> </p><p>I also agree with your second point. I had to remind myself, while re-reading my research questions, of the reasons why each was interesting. I will reflect and fill in this missing information.</p>